website/templates/flatpages/default.html

```diff
@@ -0,0 +1,9 @@
+{% extends 'base.html' %}
```

Comment:
We can also tweak the `base.html` template if you need to include other sections from flatpages (not sure if there's more needed beyond the title/content)